source/manual/fall-back-to-backup-cdn.html.md

```diff
@@ -0,0 +1,36 @@
+---
+owner_slack: "#re-govuk"
+title: Fall back to backup CDN
+section: Deployment
+layout: manual_layout
+parent: "/manual.html"
+---
+
+There two backup Content Delivery Network (CDN) that can be used if Fastly is down.
+AWS CloudFront or Google Cloud CDN.
+
+**Important** At the time of writing, GOV.UK will be served
+by static mirrors and not the origin of GOV.UK if you fallback to one of the backup CDN.
+
+## Procedure
+
+To switch to AWS Cloudfront:
+You will have to make 2 DNS changes to GOV.UK:
+
+1. you will have to ask JISC (very few people are authorised to make such changes) to change
+   the cname of `www.gov.uk` to the `www` AWS CloudFront distribution domain.
+   You can log into the AWS web console to find the `www` AWS CloudFront distribution domain.
+
+2. you will have to change the cname of `assets.publishing.service.gov.uk` to
+   the `assets` AWS CloudFront distribution domain using the usual gov.uk processes.
+   You can log into the AWS web console to find the `assets` AWS CloudFront distribution domain.
+
+To switch to Google Cloud CDN
+To access the GCP console you will neeed to [set up your GCP account](/manual/set-up-gcp-account.html)
```

Comment:
I've made a PR to the GCP account docs (#2955).

People reading these docs should already have access now that we've got the google group set up, so perhaps this doesn't need to call out the need to set  the access up anymore. Maybe something like:

> You can access the Google Cloud Platform console at [https://console.cloud.google.com/](https://console.cloud.google.com/). Everyone with production access should have access to GCP. If you don't, follow the docs to [set up your GCP account](/manual/set-up-gcp-account.html).